src/Query.php

```diff
@@ -86,6 +86,10 @@
  * @method $this setFacet( Set $facet )
  * @method $this addFields( string $fields )
  * @method $this addSort( $sorting )
+ * @method string getQueryParser( string $default = '' )
+ * @method $this setQueryParser( string $type )
+ * @method string getQueryFields( string $default = '' )
```

Comment:
Fields sollten als Array übergeben werden, oder? In der Solarium API ist es dann ein String. Das ist klar, aber an welcher Stelle wollen wir aus unserer konfigurierten Liste, ein Array, einen String machen? Gibt es einen guten Grund das weiter oben anzusiedeln?